src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1159,22 +1165,27 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 				$body[] = __( 'These plugins are now up to date:' );
 
 				foreach ( $successful_updates['plugin'] as $item ) {
+					$body_message = '';
 					if ( $item->item->current_version ) {
-						$body[] = sprintf(
+						$body_message .= sprintf(
 							/* translators: 1: Plugin name, 2: Current version number, 3: New version number. */
 							__( '- %1$s (from version %2$s to %3$s)' ),
 							$item->name,
 							$item->item->current_version,
 							$item->item->new_version
 						);
 					} else {
-						$body[] = sprintf(
+						$body_message .= sprintf(
 							/* translators: 1: Plugin name, 2: Version number. */
 							__( '- %1$s version %2$s' ),
 							$item->name,
 							$item->item->new_version
 						);
 					}
+					if ( $item->item->url ) {
+						$body_message .= sprintf( ': %s', esc_url( $item->item->url ) );
```

Comment:
I think this will need to be restructured so you have four possible strings

```
'- %1$s (from version %2$s to %3$s)'
'- %1$s (from version %2$s to %3$s) : %4$s'
'- %1$s version %2$s'
'- %1$s version %2$s : %3$s'
```

While appending the URL to the line makes sense in some languages, including English, it may not make sense in all languages so the strings need to account for that.